apps/studio/components/ui/ErrorBoundaryState.tsx

```diff
@@ -1,15 +1,33 @@
+import { isError } from 'lodash'
 import { ExternalLink } from 'lucide-react'
 import Link from 'next/link'
 import { useRouter } from 'next/router'
-import type { FallbackProps } from 'react-error-boundary'
 
-import { AlertDescription_Shadcn_, AlertTitle_Shadcn_, Alert_Shadcn_, Button } from 'ui'
-import { WarningIcon } from 'ui'
+import {
+  AlertDescription_Shadcn_,
+  AlertTitle_Shadcn_,
+  Alert_Shadcn_,
+  Button,
+  WarningIcon,
+} from 'ui'
+
+// More correct version of FallbackProps from react-error-boundary
+export type FallbackProps = {
+  error: unknown
+  resetErrorBoundary: (...args: any[]) => void
+}
 
 export const ErrorBoundaryState = ({ error, resetErrorBoundary }: FallbackProps) => {
   const router = useRouter()
-  const message = `Path name: ${router.pathname}\n\n${error.stack}`
-  const isRemoveChildError = error.message.includes("Failed to execute 'removeChild' on 'Node'")
+
+  let errorMessage = ''
+  let urlMessage = ''
+  let isRemoveChildError = false
+  if (isError(error)) {
+    errorMessage = error.message
```

Comment:
could we use ternary for this? in favor of immutability

`const errorMessage = isError(error) ? error.message ?? ''`